api/v1alpha1/drpolicy_types.go

```diff
@@ -71,7 +71,7 @@ type DRPolicySpec struct {
 	// minutes, 'h' means hours and 'd' stands for days.
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern=`^\d+[mhd]$`
-	SchedulingInterval string `json:"schedulingInterval"`
+	SchedulingInterval string `json:"schedulingInterval,omitempty"`
```

Comment:
These [kubebuilder annotations](https://book.kubebuilder.io/reference/markers/crd-validation.html) are intended to [validate](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation) the value of the field on the API server before persisting the object instead of writing controller code to validate it after it has been persisted.  I'm not sure how to preserve this behavior.

I wonder if the field could alternatively be interpreted as a recovery point object, perhaps even renamed, such that it would make sense to set it to zero for sync/metro DR.

Reply:
> I wonder if the field could alternatively be interpreted as a recovery point objective, perhaps even renamed, such that it would make sense to set it to zero for sync/metro DR.

In the (near?) future we should start supporting a common policy for both sync and async, and depending on the cluster relationships we should be able to failover with 0 RPO or ~schedulingInterval RPO. So choosing a 0 value into the future would not be useful, but we do require some thought on how to adapt to both requirements.